flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/SchemaChangeHelper.java

```diff
@@ -0,0 +1,109 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.flink.sink.writer;
+
+import org.apache.doris.flink.catalog.doris.FieldSchema;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.compress.utils.Lists;
+import org.apache.flink.util.StringUtils;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class SchemaChangeHelper {
+    private static final List<String> dropFieldSchemas = Lists.newArrayList();
+    private static final List<FieldSchema> addFieldSchemas = Lists.newArrayList();
+    // Used to determine whether the doris table supports ddl
+    private static final List<DDLSchema> ddlSchemas = Lists.newArrayList();
+    public static final String ADD_DDL = "ALTER TABLE %s ADD COLUMN %s %s";
+    public static final String DROP_DDL = "ALTER TABLE %s DROP COLUMN %s";
```

Comment:
only saw add and drop column, how is rename implemented?

Reply:
Rename is currently not supported, the rename statement will be filtered, and doris will not execute it.